src/config.js

```diff
@@ -0,0 +1,18 @@
+import Utils from './utils';
+
+if (!window.prismic || !window.prismic.endpoint) throw new Error('Prismic.js requires window.prismic.endpoint');
+
+const PRISMIC_ENDPOINT = window.prismic.endpoint.replace(/\.cdn\.prismic\.io/, '.prismic.io');
+const matches = PRISMIC_ENDPOINT && PRISMIC_ENDPOINT.match(new RegExp('https?://([^/]*)'));
+
+export default matches ? {
+  baseURL: matches[0],
+  editorTab: matches[1],
+  corsLink: Utils.iFrame(`${matches[0]}/previews/messenger`),
```

Comment:
I don't it's unexpected to have `dom element` in the `config`. You could do this directly in `share.js`.